Library/RSBot.Core/Objects/Player.cs

```diff
@@ -739,6 +739,22 @@ public bool UsePurificationPill()
             return result;
         }
 
+        /// <summary>
+        /// Uses the speed drug.
+        /// </summary>
+        /// <returns></returns>
+        public bool UseSpeedDrug()
+        {
+            if (State.LifeState == LifeState.Dead)
+                return false;
+
+            var item = Game.Player.Inventory.GetItems(new TypeIdFilter(3, 3, 13, 1)).Find(p => p.Record.Desc1.Contains("_SPEED_"));
```

Comment:
Please be careful if you are using CodeName128 values, since they may be different from server to server. Try to get it by a matching Param or smth like that.

Reply:
Well, it's how it's always been used, I just moved this piece of code from one place to another. We could create an issue for finding a better way to get the speed drug from player's inventory.